.gitignore

```diff
@@ -0,0 +1,12 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.venv/
+.pytest_cache/
+**__pycache__/
+*.pyc
+**db.sqlite3
+tests
```

Comment:
The `tests` directory is included in the `.gitignore` file. If you plan to add test files to your repository, consider removing this line to ensure that your test files are tracked by version control.